### DISCUSSION TITLE
Custom plots

### DISCUSSION THREAD

**Discussion by ogreyesp** [getredash/redash#5938] [General]
Hi

I'm interested in using Redash for creating Data Visualization dashboards. I would like to run custom datasources written in Python (as described here https://medium.com/geekculture/re-dash-python-data-source-e5200a863cb2) and show custom plots (.e.g by using plotly) in the dashboards.

Is it possible to show custom plots (it is not necessary to modify them via UI) in Redash?

**Comment by arikfr** [comment]
By custom plots you mean ones defined by the Python code? In this case, then no.

**Comment by elhamgetahun** [comment]
Hi everyone,

I am currently working on a dashboard that uses Oracle DB and Elasticsearch as data sources. I would like to implement a custom visualization using CSS and JavaScript, but I'm facing challenges in achieving this.

Could anyone guide me or provide resources on how to create custom visualizations in Redash?

Thank you in advance for your help!
